### DISCUSSION TITLE
douyin.com: block login screen

### DISCUSSION THREAD

**Discussion by 3xploiton3** [realodix/AdBlockID#1570] [Personal Filter]
https://www.douyin.com/

cara filter / block login screen bagaimana?

saya coba pakai tools element picker UO

> ! 2023-10-04 https://www.douyin.com
www.douyin.com###login-pannel
||p-pc-weboff.byteimg.com/tos-cn-i-9r5gewecjs/test.png$image
www.douyin.com###login-full-panel-3uhxs9y3k9e0
www.douyin.com###login-full-panel-2243g6urjtj4

terkadang masih muncul

**Comment by realodix** [comment]
```adblock
douyin.com##body > [id^="login-full-panel-"]
```

> terkadang masih muncul

Itu karena `login-full-panel-[karakter acak selalu berganti]`

**Comment by 3xploiton3** [comment]
sudah ketemu pakai filter ini

> douyin.com##[id^="login-full-panel"]
